pipelines/data_embedding_pipeline/data_embedding_pipeline.py

```diff
@@ -16,5 +16,5 @@ def data_embedding_pipeline() -> None:
     """
     mind_df, nhs_df = fetch_data()
 
-    mind_df = embed_data(mind_df)
-    nhs_df = embed_data(nhs_df)
+    embed_data(mind_df, "base", "v1.0", "mind_data")
```

Comment:
Is it possible for these to be parameterised in the config? Specifically the `embed_model_type` and `data_version`?

Reply:
I have added both to the configuration, i am not sure how we use these in the pipeline.